packages/flame-css/README.md

```diff
@@ -0,0 +1,36 @@
+# Flame css
+
+`flame-css` is an alternative offering to `flame`.
+
+It offers a sass codebase and an easy to consume css file.
+
+@TODO:
+
+- [ ] Hook flame-tokens/flame-foundations as primary variables
+- [ ] Integrate tailwind to output utility classes
+- [x] Storybook setup
+- [x] Write all stories
+- [ ] Snapshot testing
+- [ ] Publishing
```

Comment:
Publishing will be done as soon as we merge/`yarn release` right?

Reply:
gonna remove that particular TODO as it was only pertinent for my personal tracking.